addons/base_sparse_field/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+
+from . import fields
+from . import models
```

Comment:
Did you realize that due to #8591 monkey patching in fields file is going to happen no matter the add-on is installed or not? Then why split it?